frontend/src/data/actions.js

```diff
@@ -4,6 +4,8 @@ export const types = {
   ERROR: "ERROR",
   ADDUSER: "ADDUSER",
   DOWNLOADIMAGE: "DOWNLOADIMAGE",
```

Comment:
It's actually so minor tho, but is it okay if we change it to DOWNLOAD_IMAGE and then GET_WORDS/GET_WORDS_SUCCESS -> it's just easier to read in console for debugging

Reply:
I am going to be removing DOWNLOADIMAGE in a followup pr, so I will keep that for now, but will change GET_WORDS and GET_WORDS_SUCCESS